engines/ultima/ultima8/metaengine.cpp

```diff
@@ -41,89 +41,89 @@ struct KeybindingRecord {
+	{ ACTION_STEP_BACK, "STEP_BACK", _s("Step back"), "END", "KP2" },
+	{ ACTION_ROLL_LEFT, "ROLL_LEFT", _s("Roll left"), "INSERT", "KP1" },
+	{ ACTION_ROLL_RIGHT, "ROLL_RIGHT", _s("Roll right"), "PAGEUP", "KP3" },
+	{ ACTION_TOGGLE_CROUCH, "TOGGLE_CROUCH", _s("Toggle crouch"), "KP5", nullptr },
+	{ ACTION_MOVE_STEP, "MOVE_STEP", _s("Side step / Advance / Retreat"), "LALT", "JOY_RIGHT_SHOULDER" },
 	{ ACTION_NONE, nullptr, nullptr, nullptr, nullptr }
 };
 
 static const KeybindingRecord CHEAT_KEYS[] = {
-	{ ACTION_CLIPPING, "CLIPPING", _s("Toggle Clipping"), "M+INSERT", nullptr },
+	{ ACTION_CLIPPING, "CLIPPING", _s("Toggle clipping"), "M+INSERT", nullptr },
 	{ ACTION_QUICK_MOVE_ASCEND, "ASCEND", _s("Ascend"), "M+HOME", nullptr },
 	{ ACTION_QUICK_MOVE_DESCEND, "DESCEND", _s("Descend"), "M+END", nullptr },
-	{ ACTION_QUICK_MOVE_UP, "MOVE_UP", _s("Move Up"), "M+UP", nullptr },
-	{ ACTION_QUICK_MOVE_DOWN, "MOVE_DOWN", _s("Move Down"), "M+DOWN", nullptr },
-	{ ACTION_QUICK_MOVE_LEFT, "MOVE_LEFT", _s("Move Left"), "M+LEFT", nullptr },
-	{ ACTION_QUICK_MOVE_RIGHT, "MOVE_RIGHT", _s("Move Right"), "M+RIGHT", nullptr },
+	{ ACTION_QUICK_MOVE_UP, "MOVE_UP", _s("Move up"), "M+UP", nullptr },
+	{ ACTION_QUICK_MOVE_DOWN, "MOVE_DOWN", _s("Move down"), "M+DOWN", nullptr },
+	{ ACTION_QUICK_MOVE_LEFT, "MOVE_LEFT", _s("Move left"), "M+LEFT", nullptr },
+	{ ACTION_QUICK_MOVE_RIGHT, "MOVE_RIGHT", _s("Move right"), "M+RIGHT", nullptr },
 
 	{ ACTION_NONE, nullptr, nullptr, nullptr, nullptr }
 };
 
 #ifndef RELEASE_BUILD
 static const KeybindingRecord DEBUG_KEYS[] = {
-	{ ACTION_TOGGLE_TOUCHING, "TOUCHING", _s("Show Touching Items"), "C+h", nullptr },
-	{ ACTION_TOGGLE_PAINT, "TOGGLE_PAINT", _s("Show Editor Items"), "F9", nullptr },
-	{ ACTION_DEC_SORT_ORDER, "DEC_SORT_ORDER", _s("Decrement Map Sort Order"), "LEFTBRACKET", nullptr },
-	{ ACTION_INC_SORT_ORDER, "INC_SORT_ORDER", _s("Increment Map Sort Order"), "RIGHTBRACKET", nullptr },
-	{ ACTION_FRAME_BY_FRAME, "FRAME_BY_FRAME", _s("Toggle Frame By Frame"), "F12", nullptr },
-	{ ACTION_ADVANCE_FRAME, "ADVANCE_FRAME", _s("Advance Frame"), "f", nullptr },
-	{ ACTION_TOGGLE_STASIS, "TOGGLE_STASIS", _s("Toggle Avatar In Stasis"), "F10", nullptr },
-	{ ACTION_SHAPE_VIEWER, "SHAPE_VIEWER", _s("Show Shape Viewer"), "F11", nullptr },
+	{ ACTION_TOGGLE_TOUCHING, "TOUCHING", _s("Show touching items"), "C+h", nullptr },
+	{ ACTION_TOGGLE_PAINT, "TOGGLE_PAINT", _s("Show editor items"), "F9", nullptr },
+	{ ACTION_DEC_SORT_ORDER, "DEC_SORT_ORDER", _s("Decrement map sort order"), "LEFTBRACKET", nullptr },
+	{ ACTION_INC_SORT_ORDER, "INC_SORT_ORDER", _s("Increment map sort order"), "RIGHTBRACKET", nullptr },
+	{ ACTION_FRAME_BY_FRAME, "FRAME_BY_FRAME", _s("Toggle frame by frame"), "F12", nullptr },
+	{ ACTION_ADVANCE_FRAME, "ADVANCE_FRAME", _s("Advance frame"), "f", nullptr },
+	{ ACTION_TOGGLE_STASIS, "TOGGLE_STASIS", _s("Toggle avatar in stasis"), "F10", nullptr },
```

Comment:
"Avatar" or "The Avatar" is the term for the player character in Ultima games. It should probably remain capitalized.